src/exercise/03.md

```diff
@@ -2,6 +2,9 @@
 
 ## 📝 Your Notes
 
+- Interpolation is how we would generally handle this in production.
+- I've had props spread beaten out of me in code review...
```

Comment:
oof yeah i totally agree here. the readability is a lot better with interpolation imo. good choice leaving it as a comment!